app_dart/test/src/datastore/fake_datastore.dart

```diff
@@ -107,7 +107,10 @@ class FakeDatastoreDB implements DatastoreDB {
 
   @override
   FakeQuery<T> query<T extends Model<dynamic>>({Partition partition, Key<dynamic> ancestorKey}) {
-    final List<T> results = values.values.whereType<T>().toList();
+    List<T> results = values.values.whereType<T>().toList();
+    if (ancestorKey != null) {
```

Comment:
I noticed this while debugging we only return by type, so I extended it to filter the ancestor key (none of the tests take advantage of this yet)